Permissions for component owner and signee

den-gts opened the pull request:
**Component owner** is user who has permission "panopticum.change_owner_status". CO can edit own component versions except _signee status_ and _signee notes_. If component version maintainer does not match to current user then that component version not available for editing.

**Signee** is user who has permission "panopticum.change_signee_status".  Signee able add\change signee status and signee notes for all component versions. Signee need not to have permission "change component version". Signee available to view and edit only requirements that exist in owned requirement sets.

Commit:
add permissions for component owner and signee

Commit:
Add support filtering for component version at requirement

Commit:
add requirement set owner group

Commit:
Allow change requirements by signee and disallow change not owned component
versions

Commit:
Allow change requirements by signee and disallow change not owned component
versions

Commit:
Merge remote-tracking branch 'origin/feature/perm' into feature/perm

# Conflicts:
#	panopticum/admin.py

perfguru87 commented on panopticum/admin.py:
```diff
@@ -87,50 +89,72 @@ class RequirementForm(django.forms.ModelForm):
     from requirement statues by django admin form and frontend """
     owner_status = panopticum.fields.RequirementStatusChoiceField(queryset=RequirementStatus.objects.all(),
-                                                           label='Readiness')
+                                                                  label='Readiness')
```
tab?

den-gts replied:
Whats wrong? That change just align ```label='Readiness``` to ```queryset=...```

perfguru87 replied:
I'm talking about pep8 (tab vs whitespaces)
minor thing

perfguru87 commented on panopticum/admin.py:
```diff
@@ -87,50 +89,72 @@ class RequirementForm(django.forms.ModelForm):
+            'owner_status': 'Readiness',
+            'owner_notes': 'Notes',
+            'approve_status': 'Sign off',
+            'approve_notes': 'Sign off notes'
```
mb "Signee notes"?

den-gts replied:
will replace to ```Signee notes```

perfguru87 commented on panopticum/fixtures/init.json:
```diff
@@ -169,6 +169,22 @@
+      "title": "FIPS compliance",
+      "description": ""
+    }
+  },
```
GDPR compliance is missing, it's one of the most important one

den-gts replied:
It's public compliance?

Commit:
Fix issue with forbidden change component version for signee.
Workaround: signee must has permission for change component version and
change signee status. "Change signee status" permission will set readonly
for all non-requirements fields

Commit:
add GDPR compliance to fixture init.json
